src/App.js

```diff
@@ -1,11 +1,11 @@
 import React, { Component } from "react";
-import MentorImage from "./images/mentor.jpg";
+import mentorImage from "./images/mentor.jpg";
```

Comment:
👍

Reply:
Surprisingly, this has not been approved yet for development branch. 
@emmawedekind don't you think it's a good practice to use constants? I believe it makes the code even more clean.